docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
+
+**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+
+In Filecoin, users pay storage miners to store their files. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
```

Comment:
The combo of the first two sentences makes it sound like I'm paying a computer, not a human or a company that owns computers.

Reply:
Good call. I tried a few different wordings here and settled on “users pay to store their files on storage miners”. I decided that it would be better to leave the distinction between storage miner/storage provider until later down the page.